service/systemd/service.go

```diff
@@ -73,8 +76,18 @@ type Service struct {
 	newDBus DBusAPIFactory
 }
 
-// NewService returns a new value that implements Service for systemd.
-func NewService(name string, conf common.Conf, dataDir string, newDBus DBusAPIFactory, fallBackDirName string) (*Service, error) {
+// NewServiceWithDefaults returns a new systemd service reference populated
+// with sensible defaults.
+func NewServiceWithDefaults(name string, conf common.Conf) (*Service, error) {
+	svc, err := NewService(name, conf, DataDir, NewDBusAPI, renderer.Join(paths.NixDataDir, "init"))
```

Comment:
Wondering about paths.DataDir here too instead of NixDataDir.

Reply:
One of steps to break cyclic dependency if/when the service manager is relocated, is for it to instantiate systemd services directly instead of asking the service package for them. This is a design flaw - a detail depending on its parent abstraction.

So we push the knowledge of how to instantiate a typical service into its implementation and get it from there. And its the same as the other changes - we no longer use series to get OS, to get something else (init system, file path) when we know the OS.

I am confident that there is no logic calling `paths.[Log|Data]Dir(series)` that will return a different result than the direct access under systemd.